src/install.rs

```diff
@@ -136,3 +150,95 @@ fn chmod_x(file: &PathBuf) {
+    io::stderr().write_all(&output.stderr).unwrap();
+
+    // delete the zip file
+    info!("  Deleting zip file {}", archive_path.clone());
+    if let Err(e) = fs::remove_file(archive_path) {
+        error!("Failed to remove archive. {}", e)
+    };
+
+    info!("Mac zip postprocessing done...");
+}
+
+fn mac_process_dmg(archive_path: String) {
+    info!("Mac dmg postprocessing starting...");
+    
+    // Mount the disk image file
+    info!("  Mounting dmg file {}", archive_path.clone());
+    let output = Command::new("/usr/bin/hdiutil")
+                            .arg("attach")
+                            .arg(archive_path.clone())
+                            .output()
+                            .expect("Mount failed");
+    info!("  Result of mount: {}", output.status);
+    io::stdout().write_all(&output.stdout).unwrap();
+    io::stderr().write_all(&output.stderr).unwrap();
+
+    // we need the stem of the archive name, because this is the name under which MacOS mounts
+    // and the target is the instance location
+    let mount_name = Path::new(&archive_path).file_stem().unwrap().to_str().unwrap();
+    let archive_parent = Path::new(&archive_path).parent().unwrap().to_str().unwrap();
+
+    // Now create a script, becase it's not possible to do this from the rust runtime directly (MacOS security)
+    let buffer = "#!/bin/sh\n\nsource=\"$1\"\ntarget=\"$2\"\n\ncp -r \"$source\" \"$target\"\n";
+    let script_path = format!("{}/ESLauncher2.sh", archive_parent.clone());
+    let mut script_file = File::create(script_path.clone()).expect("Creation of script failed!");
+    if let Err(e) = script_file.write_all(buffer.as_bytes()) {
+        error!("Failed to write script. {}", e)
+    };
+    
+    // Call the script
+    let app_source_path = format!("/Volumes/{}/Endless Sky.app", mount_name);
+    let app_target_path = format!("{}/", archive_parent);
+    info!("  Calling copy script with parameters:");
+    info!("    {}", app_source_path.clone());
+    info!("    {}", app_target_path.clone());
+    let output = Command::new("./ESLauncher2_copy.sh")
+                            .arg(app_source_path.clone())
+                            .arg(app_target_path.clone())
+                            .output()
+                            .expect("Copy failed");
+    info!("  Result of copy: {}", output.status);
+    io::stdout().write_all(&output.stdout).unwrap();
+    io::stderr().write_all(&output.stderr).unwrap();
+
+    // detach the drive
+    let detach_path = format!("/Volumes/{}", mount_name);
+    info!("  Detaching {}", detach_path.clone());
+    let output = Command::new("/usr/bin/hdiutil")
+                            .arg("detach")
+                            .arg(detach_path.clone())
+                            .output()
+                            .expect("Detach failed");
+    info!("  Result of detach: {}", output.status);
+    io::stdout().write_all(&output.stdout).unwrap();
```

Comment:
Whereever possible, use the `log` macros - stdout is but one console which shouldn't even be shown by default. The loggers also write the logfile. As for logging some things to the log file *only* (without showing them in the UI), that's a pending problem.

Reply:
Will search for a usage of the logger. I picked this up in a spot and thought that is how logging is done...